db/jksndb.sql

```diff
@@ -0,0 +1,217 @@
+CREATE TABLE IF NOT EXISTS `jksndb`.`company` (
+  `id` INT NOT NULL,
+  `name` VARCHAR(45) NULL,
+  `Address1` VARCHAR(45) NULL,
+  `Address2` VARCHAR(45) NULL,
+  `cell_phone` VARCHAR(45) NULL,
+  `office_phone` VARCHAR(45) NULL,
+  `City` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`role`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`role` (
+  `id` INT NOT NULL,
+  `role` INT NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`user`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`user` (
+  `username` (16) NOT NULL,
+  `email` (255) NULL,
+  `password` (32) NOT NULL,
+  `create_time`  NULL DEFAULT CURRENT_TIMESTAMP,
+  `id` INT NOT NULL,
+  `wechat_id` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `id`
+    FOREIGN KEY ()
+    REFERENCES `jksndb`.`company` ()
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `id`
+    FOREIGN KEY ()
+    REFERENCES `jksndb`.`role` ()
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION);
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`vendor`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`vendor` (
+  `id` INT NOT NULL,
+  `name` VARCHAR(45) NULL,
+  `city` VARCHAR(45) NULL,
+  `cell_phone` VARCHAR(45) NULL,
+  `office_phone` VARCHAR(45) NULL,
+  `address1` VARCHAR(45) NULL,
+  `address2` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`category`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`category` (
+  `id` INT NOT NULL,
+  `discription` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`equip`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`equip` (
+  `id` INT NOT NULL,
+  `model` VARCHAR(45) NULL,
+  `sn` VARCHAR(45) NULL,
+  `discription` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `id`
+    FOREIGN KEY ()
+    REFERENCES `jksndb`.`vendor` ()
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `id`
+    FOREIGN KEY ()
+    REFERENCES `jksndb`.`category` ()
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`order`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`order` (
+  `id` INT NOT NULL,
+  `signtime` DATETIME NULL,
+  `starttime` DATETIME NULL,
+  `endtime` DATETIME NULL,
+  `duration` INT NULL,
+  `money` INT NULL,
+  PRIMARY KEY (`id`),
```

Comment:
suggest to add more 'stauts' to save the order stauts. that would be userful for search/filter